pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/kit/KitTypeImpl.java

```diff
@@ -31,14 +38,57 @@ public class KitTypeImpl implements KitTypeDao {
             + "kit_type "
             + "WHERE kit_type_id = ?";
 
+    private static final String SQL_SELECT_KIT_TYPE = "SELECT * FROM ddp_kit_request_settings kitSettings "
+            + "LEFT JOIN kit_type type ON (type.kit_type_id = kitSettings.kit_type_id) "
+            + "LEFT JOIN ddp_instance realm ON (realm.ddp_instance_id = kitSettings.ddp_instance_id) "
+            + " WHERE realm.instance_name = ? AND type.kit_type_name = ?";
+
+    /**
+     * Getting kitTypes based on kit type name and realm
+     */
+    public static KitTypeDto getKitTypes(@NonNull String realm, @NonNull String kitTypeName) {
+        log.info("Finding kit typ for realm {} with kit type name {}", realm, kitTypeName);
+        List<KitTypeDto> kitTypes = new ArrayList<>();
+        SimpleResult results = inTransaction((conn) -> {
+            SimpleResult dbVals = new SimpleResult();
```

Comment:
SimpleResult is probably not needed since you are filling `kitTypes` directly. You also handle exceptions directly.